api/src/API/Controllers/OrganizationsController.cs

```diff
@@ -43,11 +49,15 @@ public async Task<IActionResult> GetOneOrganization(string orgId)
         }
 
         [HttpPost]
-        public async Task<IActionResult> CreateNewOrganization(Organization organization)
+        public async Task<IActionResult> CreateNewOrganization([FromBody] CreateOrganizationRequest body)
         {
-            organization.Id = Guid.NewGuid();
+            var organization = body.Organization;
```

Comment:
Why are we accepting whatever value is passed in here?

Reply:
ID gets assigned in AddOneAsync